### PR TITLE
XRT-1900: Update mqtt config - KeepAliveInterval and RetryInterval

### PR DESCRIPTION
**Pull request by BinduRao2018**
- KeepAliveInterval is set to 60 secs, which is recommended
- Update MQTT Config - RetryInterval is read as part of client config. Unused configuration variable Retrycount deleted.

### CODE REVIEW

File: Exporters/mqtt/mosquitto/deployment/config/mqtt_bridge.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "QoS": 1,
     "ClientConfig":
     {
-      "KeepAliveInterval": 0,
+      "KeepAliveInterval": 60,
       "MQTTVersion": 4,
```
Comment (FraserG01) [inline Exporters/mqtt/mosquitto/deployment/config/mqtt_bridge.json:13]: does this not need a retry interval as well?

**Review by FraserG01 — CHANGES_REQUESTED** [review]
one minor point of change

File: Exporters/mqtt/mosquitto/deployment/config/mqtt_bridge.json (hunk 1 above)
Comment (BinduRao2018) [reply Exporters/mqtt/mosquitto/deployment/config/mqtt_bridge.json:13]: 'RetryInterval' is an optional configuration, which defaults to 5 secs. I have retained in those configs that already had them

**Review by FraserG01 — APPROVED** [review]
LGTM

**Pull request merged**